src/modules/person/persistence/rolePermission.entity.ts

```diff
@@ -0,0 +1,14 @@
+import { TimestampedEntity } from './timestamped.entity.js';
+import { Entity, ManyToOne, PrimaryKey } from '@mikro-orm/core';
+import { PermissionEntity } from './permission.entity.js';
+
+@Entity({
+    tableName: 'role_permission',
+})
+export class RolePermissionEntity extends TimestampedEntity<RolePermissionEntity, 'id'> {
+    @PrimaryKey()
+    public readonly id!: string;
+
+    @ManyToOne()
+    public permission!: PermissionEntity;
```

Comment:
ServiceProviderAccess does not have a permission, but extends this entity.
From my point fo view it can be modeled in two ways:
1. RolePermission has a reference to either Permission or ServiceProviderAccess
2. RolePermission is the super class for Permission and ServicProvierAccess
I think the mixture of both won't work.

Reply:
Permission does now no longer exist. We'll call it ServiceAccess and extend RolePermission